### DISCUSSION TITLE
ZFS on Windows 2.2.3 problems

### DISCUSSION THREAD

**Discussion by guenther-alka** [openzfsonwindows/openzfs#362] [General]
I installed 2.2.3rc1 on Windows 11 with the "add path" option enabled during setup.
After reboot zfs and zpool were not found in a terminal

The following command fixed it.
$env:path +='C:\Program Files\OpenZFS On Windows'

**Comment by guenther-alka** [comment]
As I had some problems with my client server app, I decided to reinstall Open-ZFS 2.2.3rc1
At the end of the installation process (Windows 11 Pro 23H2 german), Windows crashes. I supppose this was also the reason of the missing path problem. I tried to install again with same result. I tried to deinstall but there was no entry in System settings > Apps

When I try to start "C:\Program Files\OpenZFS On Windows\unins000.exe" I get the message "setup files are corrupted, get a new copy." 

registry settings HKLM:\Software\OpenZFS (not created)
registry settings HKLM:\System\ControlSet001\Services\OpenZFS (created)

**Comment by lundman** [comment]
Yeah something is being weird, I'll remove the 2.2.3rc1 for now, until I can figure out why it dies after creating a thread.

**Comment by lundman** [comment]
OK add path is just a side-effect of it crashing before setting path. the crash has been addressed in rc2. ad3d9f7

**Comment by besterino** [comment]
I installed rc2-3 on Server2022 and Win11 (fresh installs) without any issues. Path variable also works.
